pages/[slug].js

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useRef, useEffect } from "react";
+import React, { useState, useRef, useEffect, useLayoutEffect } from "react";
```

Comment:
Keep the height of the index auto or add scroll, otherwise user will not have idea about scrolling in index